package.json

```diff
@@ -1,16 +1,21 @@
 {
   "name": "bm-tslint-rules",
-  "version": "0.0.8",
+  "version": "0.1.0",
   "description": "default tslint rules to use for projects using typescript",
+  "repository": "https://github.com/bettermarks/bm-tslint-rules",
+  "bugs": "https://github.com/bettermarks/bm-tslint-rules/issues",
   "main": "tslint.json",
   "files": [
     "dist"
   ],
   "engines": {
-    "node": ">=8.1.4",
-    "npm": ">=5.0.3"
+    "node": ">=8.2.0",
```

Comment:
Question: Since `.nvmrc` sets node version to `8.9.0` and it's LTS version, I propose that we could this to be same.

Reply:
in `.nvmrc` we are setting the version to easily use for development.
In `package.json` we are setting the minimal required version for installing/using the lib.
So as long as we don't know about any incompatibilities I think it is fine to stick the the minimum that we need (in this case we need it because we are using `npx` which was first available in node 8.2/npm 5.2.
Which is also the reason we are still using that version on CIrcleCI to prove it still works with this version.
I also tried to explain this in the PR description, but maybe it was hard to spot there.